pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -249,6 +284,13 @@ var _ = Describe("ImageUpload", func() {
 			Expect(cmd()).NotTo(BeNil())
 		})
 
+		It("uploadProxyURL not configured", func() {
```

Comment:
... this test look pretty similar. What is the difference?

Reply:
oops, forgot to update this one.